tests/test_config_validation/test_validation_step.py

```diff
@@ -310,128 +312,42 @@
         run:
           services:
             my-service-container:
-              # The 'build' attribute functions the same way that the step
```

Comment:
I'm guessing this was originally taken from an example in the documentation ?

Reply:
That was my guess as well